[2024-01-15T15:01:51.253-0300] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cities_novo.df scheduled__2024-01-15T17:59:40.621620+00:00 [queued]>
[2024-01-15T15:01:51.255-0300] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cities_novo.df scheduled__2024-01-15T17:59:40.621620+00:00 [queued]>
[2024-01-15T15:01:51.256-0300] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-01-15T15:01:51.261-0300] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): df> on 2024-01-15 17:59:40.621620+00:00
[2024-01-15T15:01:51.265-0300] {standard_task_runner.py:57} INFO - Started process 66290 to run task
[2024-01-15T15:01:51.268-0300] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'cities_novo', 'df', 'scheduled__2024-01-15T17:59:40.621620+00:00', '--job-id', '1073', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpef58bs7b']
[2024-01-15T15:01:51.268-0300] {standard_task_runner.py:85} INFO - Job 1073: Subtask df
[2024-01-15T15:01:51.287-0300] {task_command.py:416} INFO - Running <TaskInstance: cities_novo.df scheduled__2024-01-15T17:59:40.621620+00:00 [running]> on host desktop
[2024-01-15T15:01:51.314-0300] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='datasetpibic@gmail.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='cities_novo' AIRFLOW_CTX_TASK_ID='df' AIRFLOW_CTX_EXECUTION_DATE='2024-01-15T17:59:40.621620+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-15T17:59:40.621620+00:00'
[2024-01-15T15:01:51.791-0300] {logging_mixin.py:154} INFO - Called | 1705341711.788654 | df
[2024-01-15T15:01:51.798-0300] {logging_mixin.py:154} INFO - SAVING FILE df
[2024-01-15T15:01:51.798-0300] {python.py:194} INFO - Done. Returned value was: None
[2024-01-15T15:01:51.801-0300] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=cities_novo, task_id=df, execution_date=20240115T175940, start_date=20240115T180151, end_date=20240115T180151
[2024-01-15T15:01:51.880-0300] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-01-15T15:01:51.889-0300] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
